src/page-templates/topic-content-type-page/topic-content-type-page-template.tsx

```diff
@@ -96,13 +102,23 @@ export const TopicContentTypePageTemplate: NextPage<
     );
 
     const [pageTitle, setPageTitle] = useState(buildPageTitle(pageNumber));
+    const [metaDescr, setMetaDescr] = useState(
+        description && description !== ''
```

Comment:
i could be wrong, but could you get away with something like `useState(description || getMetaDescr(...))`